run-tests.php

```diff
@@ -1188,6 +1188,9 @@ function system_with_timeout($commandline, $env = null, $stdin = null, $captureS
 	}
 	if ($stat["exitcode"] > 128 && $stat["exitcode"] < 160) {
 		$data .= "\nTermsig=" . ($stat["exitcode"] - 128) . "\n";
+	} else if (defined('PHP_WINDOWS_VERSION_MAJOR') && (($stat["exitcode"] >> 28) & 0b1111) === 0b1100) {
```

Comment:
Use `IS_WINDOWS`.

Reply:
It's not available prior to 8.0.

Reply:
It's defined above :)

Reply:
ctrl+f here: https://github.com/php/php-src/blob/992960e38eca2fd46601a6f286ef1908ca802e84/run-tests.php
This PR targets 7.3, `IS_WINDOWS` is not defined here. It's only defined in 8.0+'s version of `run-tests.php`: https://github.com/php/php-src/blob/PHP-8.0/run-tests.php#L155